fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalAggregationOperator.java

```diff
@@ -145,20 +145,25 @@ public DataSkewInfo getDistinctColumnDataSkew() {
     }
 
     public boolean checkGroupByCountDistinct() {
-        if (groupingKeys.size() != 1 || aggregations.size() != 1) {
+        if (groupingKeys.isEmpty() || aggregations.size() != 1) {
```

Comment:
why change this? The logic seems changed

Reply:
in previous version,  the GroupByCountDistinctDataSkewEliminateRule can only applied to one-group-by-column agg. now it can applied to multi-group-by-column, so only non-group-by agg cases are not permitted.